tensorflow/c/experimental/filesystem/modular_filesystem_test.cc

```diff
@@ -91,11 +91,17 @@ class ModularFileSystemTest : public ::testing::TestWithParam<std::string> {
     env_ = Env::Default();
   }
 
+
   void SetUp() override {
-    if (mkdir(root_dir_.c_str(), 0755) != 0) {
-      int error_code = errno;
+    FileSystem* fs = nullptr;
+    Status s = env_->GetFileSystemForFile("s3://", &fs);
+    if (fs == nullptr || !s.ok()) GTEST_SKIP() << "No filesystem registered: " << s;
+
+    s = fs->CreateDir("s3://bucketvnvo" + root_dir_);
+    if (!s.ok()) {
+      printf("Failed %s\n", s.error_message().c_str());
```

Comment:
This is experimental for now, right?

Otherwise it forces all testers to have the s3 filesystem plugin loaded before testing other plugins.

Reply:
the PR is here
https://github.com/tensorflow/tensorflow/pull/38774